### PR TITLE
Create get involved guide

### PR DESCRIPTION
**Pull request by makeasnek**
Create a get involved guide and add it to the menu

### CODE REVIEW

File: guides/get-involved.htm [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+				<p>You can help people learn about Gridcoin online and help amplify the voices of people who are mentioning Gridcoin. Please remember to always be courteous in your online engagements. Nobody likes a spammer, so be sure to respect the rules of wherever you're posting and not come across as one.</p>
+
+			<h5>Reddit links:</h5>
+				<p>Recent <a href="https://www.reddit.com/search/?q=gridcoin&sort=new" rel="nofollow">posts</a> and <a href="https://www.reddit.com/search/?q=gridcoin&type=comment&sort=new" rel="nofollow"">comments</a> related to Gridcoin</p>
+				<p>If you subscribe to a few of these subs, they will show up in your reddit feed as a reminder to make a quick comment to others. These are great places to tell poople about gridcoin: <a href="https://www.reddit.com/r/cpumining/" rel="nofollow">/r/cpumining</a>, <a href="https://www.reddit.com/r/cryptomining/" rel="nofollow">/r/cryptomining</a>,<a href="https://www.reddit.com/r/gpumining/" rel="nofollow">/r/gpumining</a>,<a href="https://www.reddit.com/r/laptopmining/" rel="nofollow">/r/laptopmining</a>,<a href="https://www.reddit.com/r/cryptocurrency/" rel="nofollow">/r/cryptocurrency</a> </p>
```
Comment (barton2526) [inline guides/get-involved.htm:33]: "poople" 👀

File: guides/get-involved.htm [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+			<h2>Crunch</h2>
+				<p>One of the most popular ways to get involved in Gridcoin is to crunch, which means using your computer's processing power to process data for scientific research. Check out our <a href="/guides/boinc-install.htm">solo crunching guide</a> to get started. </p>
+			<h2>Donate</h2>
+				<p>If you have GRC or fiat money you can donate, it can go a long way to helping Gridcoin's development and promotional efforts. You can donate GRC to the community wallet <code>bc3NA8e8E3EoTL1qhRmeprbjWcmuoZ26A2</code> (this is a multi-sig wallet whose funds can only be spent upon a network-wide vote). One of the easiest ways to donate is by setting up a sidestake, which means a portion of every stake you make will go to the community wallet. To setup a sidestake</p>
+					<ol>
+						<li>Go to the hamburger menu (top left of wallet) and select Help -> Debug window -> Console</li>
+						<li>Type in the following two commands followed by enter. Be sure to change 5 to the percent you want to donate</li>
+						<li><code>enablesidestaking=1</code></li>
+						<li><code>sidestake=bc3NA8e8E3EoTL1qhRmeprbjWcmuoZ26A2,5</code></li>
```
[the remark this answers is not in the document]
Comment (makeasnek) [reply guides/get-involved.htm:24]: My bad, thought it did. We have instructions for this in the solo crunching guide I'll update those as well. Fixed.

File: guides/get-involved.htm (hunk 1 above)
Comment (makeasnek) [reply guides/get-involved.htm]: 🤣🤣🤣🤣 fixed

File: guides/get-involved.htm (hunk 2 above)
[the remark this answers is not in the document]
Comment (barton2526) [reply guides/get-involved.htm:24]: You should probably also instruct to restart the wallet after editing the config

[the remark this answers is not in the document]
Comment (makeasnek) [reply guides/get-involved.htm:25]: Good point. Fixed in both PRs

**Review by barton2526 — APPROVED** [review]

**Pull request merged**